src/eos/eosEngine.js

```diff
@@ -94,22 +121,82 @@ export class EosEngine extends CurrencyEngine {
       }
     } catch (e) {
       this.log(`Error fetching height: ${JSON.stringify(e)}`)
```

Comment:
You can't use `JSON.stringify` on errors, since that just prints "{}" no matter what the error. You want to use `String(e)` instead, which gives you a nice description like "FooError: some message"